Slack&whatsapp-notifier.py

```diff
@@ -82,8 +60,6 @@ def Threshold_WHERE_statement(where_column, where_operation, where_condition, ti
     WHERE = f"WHERE {where_column} {where_operation} '{where_condition}' AND HOUR({time_column}) BETWEEN %s AND %s"

```

Comment:
WHERE statement should also be present in the persistent query. eg where status='FAILED' may be required in persistand query as well. 
Make method Threshold_WHERE_statement generic for both type.
Methods or function name should never start with a capital letter